client/src/containers/Explore.js

```diff
@@ -217,11 +239,73 @@ export const Explore = () => {
             .catch(error => console.log(error));    
     };
 
+    if (width < 840){ // mobile view
```

Comment:
There IS a more efficient way to do this since the order of the components is not different for mobile/desktop. In this case, when you're making a choice on which component to render, you can create a component that uses a ternary operator to choose which component to render in its place! A ternary operator is a shorthand for an if/else statement and you can use your "width < 840" condition here :)

```suggestion
    const AdBoxType = () => { return (width < 840) ? 
        ( <AdBoxMobile/> ) : 
        ( <AdBox/>) 
     };
```